client/components/navbar.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { BsSearch, BsCart3 } from 'react-icons/bs';
+import { SlHeart } from 'react-icons/sl';
+import { VscAccount } from 'react-icons/vsc';
+
+export default function NavBar() {
+  return (
+    <header className="header-upper py-1">
```

Comment:
When it comes time to clean up and improve your code, you will probably want to extract some components here.